docs/RELEASE_POLICY.md

```diff
@@ -82,6 +82,15 @@ also be specifying the supported kernel versions.
   and v3.1 will be patched since were the last two Firecracker releases and
   less than 6 months have passed since release time.
 
+## Release Status
+
+| Release | Release Date | Supported until | State         | Latest Patch |
+| ------: | -----------: | --------------: | :-----------: | -----------: |
+| v1.2    |   2022-11-30 |      2023-11-30 | Supported     | v1.2.0       |
+| v1.1    |   2022-05-06 |      2023-05-06 | Supported     | v1.1.4       |
+| v1.0    |   2022-01-31 |      2023-01-31 | Supported     | v1.0.2       |
```

Comment:
I don't think this is right. The [release policy](https://github.com/firecracker-microvm/firecracker/blob/main/docs/RELEASE_POLICY.md) says

>  the last two Firecracker vMAJOR.MINOR releases for up to 1 year from release date;
    any Firecracker vMAJOR.MINOR release for at least 6 months from release date;
    for each vMAJOR, the latest MINOR for 1 year since release date;

So 1.1 and 1.2 are supported until one year from release date, but 1.0 falls under the second category.

Reply:
Right! Another reason to have it as a table :)